### PR TITLE
CDAP-811: Delete app-manager scripts in favor of CLI

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: Wise/README.rst [hunk 1]
```diff
@@ -38,17 +38,27 @@ Build the application by executing::
...
+  $ ./bin/cdap-cli.sh deploy app <path-to-Wise-jar-file>
+  $ ./bin/cdap-cli.sh start flow Wise.WiseFlow
+  $ ./bin/cdap-cli.sh start workflow Wise.WiseWorkflow
+  $ ./bin/cdap-cli.sh start service Wise.WiseService
+  $ ./bin/cdap-cli.sh get flow status Wise.WiseFlow
+  $ ./bin/cdap-cli.sh get workflow status Wise.WiseWorkflow
+  $ ./bin/cdap-cli.sh get service status Wise.WiseService
+
+On Windows, run:
+ From the Standalone CDAP SDK directory, use the Command Line Interface::
```
Comment (awholegunch) [inline Wise/README.rst]: Delete this line. ("From the Standalone CDAP SDK directory, use the Command Line Interface::")

File: Wise/README.rst [hunk 2]
```diff
@@ -38,17 +38,27 @@ Build the application by executing::
...
+
+  $ ./bin/cdap-cli.sh deploy app <path-to-Wise-jar-file>
+  $ ./bin/cdap-cli.sh start flow Wise.WiseFlow
+  $ ./bin/cdap-cli.sh start workflow Wise.WiseWorkflow
+  $ ./bin/cdap-cli.sh start service Wise.WiseService
+  $ ./bin/cdap-cli.sh get flow status Wise.WiseFlow
+  $ ./bin/cdap-cli.sh get workflow status Wise.WiseWorkflow
+  $ ./bin/cdap-cli.sh get service status Wise.WiseService
+
+On Windows, run:
```
Comment (awholegunch) [inline Wise/README.rst]: On Windows, run:: [with a leading space, as the previous line "From the Standalone..."]

**Comment by awholegunch** [comment]
Two minor comments. Please fix and merge. Thanks!

**Comment by ghost** [comment]
@awholegunch Addressed all PR comments.

**Comment by awholegunch** [comment]
LGTM

File: Netlens/README.rst [hunk 3]
```diff
@@ -77,17 +77,26 @@ Build the Application jar::
...
+Check that they are running:
+ From the Standalone CDAP SDK directory, use the Command Line Interface::
 
-  bin/app-manager.sh --host [host] --action status
+  $ ./bin/cdap-cli.sh get flow status Netlens.AnalyticsFlow
+  $ ./bin/cdap-cli.sh get service status Netlens.AnomaliesCountService
+  $ ./bin/cdap-cli.sh get service status Netlens.AnomaliesService
+  $ ./bin/cdap-cli.sh get service status Netlens.CountersService
 
 Ingest sample traffic data::
```
Comment (sreevatsanraman) [inline Netlens/README.rst:101]: We are missing instructions for Windows. please add.

File: TwitterSentiment/README.rst [hunk 4]
```diff
@@ -61,17 +61,22 @@ Build the Application jar::
...
 
-Make sure they are running::
+Check that they are running:
+ From the Standalone CDAP SDK directory, use the Command Line Interface::
 
-  bin/app-manager.sh --host [host] --action status
+  $ ./bin/cdap-cli.sh get flow status TwitterSentiment.TwitterSentimentAnalysis
+  $ ./bin/cdap-cli.sh get service status TwitterSentiment.SentimentQuery
 
 Ingest sample statements::
```
Comment (sreevatsanraman) [inline TwitterSentiment/README.rst:81]: We are missing instructions for Windows. please add.

Comment (sreevatsanraman) [inline TwitterSentiment/README.rst:81]: As an example. Please see Wise app that you have modified in the same PR.

File: Netlens/README.rst (hunk 3 above)
Comment (sreevatsanraman) [inline Netlens/README.rst:101]: As an example. Please see Wise app that you have modified in the same PR.

**Pull request merged**